modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ParameterGroupArgumentResolver.java

```diff
@@ -76,10 +76,9 @@ public T resolve(OperationContext operationContext)
             T group = type.newInstance();
             for (Field parameterField : parameterFields)
             {
-                Object value = operationContext.getParameter(getAlias(parameterField));
-                if (value != null)
+                if (operationContext.hasParameter(getAlias(parameterField)))
                 {
-                    parameterField.set(group, value);
+                    parameterField.set(group, operationContext.getParameter(getAlias(parameterField)));
```

Comment:
I had to make that original change because there were some tests failures. Some extensions code is trying to access the parameters that have not been set.